digitalocean/resource_digitalocean_loadbalancer_test.go

```diff
@@ -728,6 +740,8 @@ resource "digitalocean_loadbalancer" "foobar" {
 
   enable_proxy_protocol    = true
   enable_backend_keepalive = true
+  http_idle_timeout_seconds = 60
```

Comment:
I'm wondering if it would be better to set this to a different value from the default to ensure the resource is set to the provided value.

```suggestion
  http_idle_timeout_seconds = 45
```